### ISSUE TITLE
'tuple' object has no attribute 'ndim' when using multiple inputs

### ISSUE THREAD

**Issue by minma2000** [SciSharp/Keras.NET#161]
My python trained model uses 4 inputs. When I tried to call the model to predict with Keras.Net, I supply the inputs like this: loadedModel.Predict(new List { x1, x2, x3, x4 }) where x1,2,3,4 have been defined as NDarray. I have verified each x has correct dimension as model expected. But when executing this line, it gives this error:

AttributeError : 'tuple' object has no attribute 'ndim'

[' File "C:\Users\myacct\anaconda3\lib\site-packages\keras\engine\training.py", line 1441, in predict\n x, _, _ = self._standardize_user_data(x)\n', ' File "C:\Users\myacct\anaconda3\lib\site-packages\keras\engine\training.py", line 579, in _standardize_user_data\n exception_prefix='input')\n', ' File "C:\Users\myacct\anaconda3\lib\site-packages\keras\engine\training_utils.py", line 99, in standardize_input_data\n data = [standardize_single_array(x) for x in data]\n', ' File "C:\Users\myacct\anaconda3\lib\site-packages\keras\engine\training_utils.py", line 99, in \n data = [standardize_single_array(x) for x in data]\n', ' File "C:\Users\myacct\anaconda3\lib\site-packages\keras\engine\training_utils.py", line 34, in standardize_single_array\n elif x.ndim == 1:\n'] at Python.Runtime.PyObject.Invoke(PyTuple args, PyDict kw)
at Python.Runtime.PyObject.InvokeMethod(String name, PyTuple args, PyDict kw)
at Keras.Base.InvokeMethod(String method, Dictionary2 args) at Keras.Models.BaseModel.Predict(List1 x, Nullable1 batch_size, Int32 verbose, Nullable1 steps, Callback[] callbacks)

Any ideas on how to fix this?
The keras.net version is 3.7.4.2 installed using nuget

This issue is also opened against Numpy.Net https://github.com/SciSharp/Numpy.NET/issues/61

**Comment by Nucs** [comment]
Note that the result is a tuple where you are trying to access an 'ndim' which is available on a numpy object.
Your output is a tuple, not a numpy object.
Tuples in python are very similar to arrays so my suggested course of action would be to check what objects exist inside that tuple. surely one of them is the result.

**Comment by minma2000** [comment]
Nucs, thanks for your reply but I didn't quite follow you comment. In my code I passed the list<ndarray> to predict function. I looked at Keras.cs file in Keras.net and it converts the list to a pyTuple object. I assume it should then pass it to Keras. I have the exact same functionality written in python as well and working. Keras is expecting a List of Input layers. Wherever this check happens I assume it is checking the ndim/shape of each Inputs. But this should be somewhere in Keras.net or Numpy.Net?
